sikkerhet/src/main/java/no/nav/familie/sikkerhet/EksternBrukerUtils.kt

```diff
@@ -8,7 +8,7 @@ import org.springframework.web.context.request.RequestContextHolder
 
 object EksternBrukerUtils {
 
-    const val ISSUER = "selvbetjening"
+    const val ISSUER = "idporten"
```

Comment:
Trenger vi en ny her? 
Denne brukes fra ba-soknad-api/familie-dokument og, så hvis vi trenger en ny, så burde vi nog legge til en ny
`const val ISSUER_IDPORTEN = "idporten"`

Reply:
Sant. Jeg skal fikse det her litt mer ordentlig etterhvert. Lukket PR for jeg er bare ute etter å teste litt.